actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -1019,7 +1019,7 @@ def build_options(selected, options = {})
         #      <option value="2000">2000</option>"
         #
         # If <tt>year_format</tt> option is passed
-        #  build_year_options(1998, start: 1998, end: 2000, year_format: ->year { "Heisei #{ year - 1988 }" })
+        #  build_year_options(1998, start: 1998, end: 2000)
         #  => "<option value="1998" selected="selected">Heisei 10</option>
```

Comment:
I think `year_format` option should not be passed to `build_year_options`. it doesn't deal with the option.

Reply:
True. In that case let's remove it.

Reply:
@rafaelfranca @shunichi  Looks like this whole piece of doc is no longer relevant, lines 1021-1023 ?

Otherwise this PR looks pretty good to me.  Can I help in some way to move it forward?

Reply:
This comment is so confusing.

In the comment, `option` is not `build_year_options`' argument but `DateTimeSelector.new`'s. 

Other methods like`month_name` have similar comments. But they have no argument named `options`. So their comments are less confusing.

I removed the comment about `year_format`  at 0c53554843bc51fbfa86c3678bd9cf5e741d6da9